backend/akvo/core_mobile/tests/test_mobile_form_endpoint.py

```diff
@@ -0,0 +1,121 @@
+from django.test import TestCase
+from django.test.utils import override_settings
+
+from akvo.core_node.models import Node
+
+
+@override_settings(USE_TZ=False)
+class TestMobileFormEndpoint(TestCase):
+    def setUp(self):
+        # POST NODE
+        payload = {
+            "name": "The Node",
+            "node_detail": [
+                {"code": "BLI", "name": "Bali"},
+                {"code": "DIY", "name": "Daerah Istimewa Yogyakarta"},
+            ],
+        }
+        data = self.client.post(
+            "/api/node",
+            payload,
+            content_type="application/json",
+        )
+        self.assertEqual(data.status_code, 200)
+        data = data.json()
+        self.assertEqual(data, {"message": "ok"})
+
+        self.node = Node.objects.first()
+
+        # POST FORM
+        payload = {
+            "id": 1694489640833,
+            "name": "New Form",
+            "description": "New Form Description",
+            "question_group": [{
+                "id": 1694489640834,
+                "name": "Nam ante Fusce",
+                "order": 1,
+                "repeatable": False,
+                "question": [{
+                    "id": 1694494676197,
+                    "order": 1,
+                    "questionGroupId": 1694489640834,
+                    "name": "Augue mauris tincidunt at Duis interdum",
+                    "type": "input",
+                    "required": False,
+                    "meta": False
+                }, {
+                    "id": 1694489640835,
+                    "order": 2,
+                    "questionGroupId": 1694489640834,
+                    "name": "Porta Lorem sed ante ac consectetur",
+                    "type": "cascade",
+                    "required": False,
+                    "meta": False,
+                    "api": {
+                        "endpoint": f"/api/node/{self.node.id}",
+                        "initial": 2,
+                        "list": "children"
+                    }
+                }]
+            }]
+        }
+        data = self.client.post(
+            "/api/form",
+            data=payload,
+            content_type="application/json"
+        )
+        self.assertEqual(data.status_code, 200)
+        result = data.json()
+        self.assertEqual(result, {"message": "ok"})
+
+    def test_endpoint_get_mobile_form_definition(self):
+        data = self.client.get(
+            "/api/device/form/1694489640833",
+        )
+        self.assertEqual(data.status_code, 200)
+        result = data.json()
+        self.assertEqual(result, {
+            "id": 1694489640833,
+            "name": "New Form",
+            "description": "New Form Description",
+            "defaultLanguage": "en",
+            "languages": ["en"],
+            "version": 1,
+            "cascades": [
+                "/sqlite/the_node.sqlite"
```

Comment:
how does the mobile app download this sqlite?

Reply:
Maybe, we can use WEBDOMAIN env from docker to provide the complete url ?

Reply:
Thanks, @anjarakvo let me know when the WEBDOMAIN is added

Reply:
I've just added it to the server manifest. Once this code is deployed, the WEBDOMAIN environment variable should be accessible.

Reply:
Thanks @anjarakvo